internal/image/live.go

```diff
@@ -93,6 +93,13 @@ func (img *LiveImage) InstantiateManifest(m *manifest.Manifest,
 		}
 		artifactPipeline = vmdkPipeline
 		artifact = vmdkPipeline.Export()
+	case platform.FORMAT_OVA:
+		vmdkPipeline := manifest.NewVMDK(m, buildPipeline, imagePipeline)
+		ovfPipeline := manifest.NewOVF(m, buildPipeline, vmdkPipeline)
+		archivePipeline := manifest.NewTar(m, buildPipeline, ovfPipeline, "archive")
+		archivePipeline.Format = osbuild.TarArchiveFormatOldgnu
```

Comment:
Out of curiosity, is this really required by VSphere to accept the tarball? 🤔 The reason this is used for the GCE image is that GCP really can't import the tarball otherwise.

Reply:
Actually the format in the ovf spec is the `ustar` format, but I haven't actually managed to get that working successfully :grimacing: